OpenSim/Analyses/StaticOptimizationTarget.cpp

```diff
@@ -479,7 +479,8 @@ objectiveFunc(const Vector &parameters, const bool new_parameters, Real &perform
     int na = _model->getActuators().getSize();
     double p = 0.0;
     for(int i=0;i<na;i++) {
-        p +=  pow(fabs(parameters[i]),_activationExponent);
+        if (ScalarActuator* act = dynamic_cast<ScalarActuator*>(&_model->getActuators().get(i)))
```

Comment:
This is part of a greater issue where people have to do these dynamic casts to ScalarActuator everywhere.